src/Test/Http.elm

```diff
@@ -22,9 +22,36 @@ These are ways to easily make `Http.Response` values for use with [`TestContext.
 import Dict exposing (Dict)
 import Expect exposing (Expectation)
 import Http
+import Json.Decode
 import SimulatedEffect
 
 
+{-| A convenient way to check something about the request body of a pending HTTP request.
+
+    ...
+        |> TestContext.assertHttpRequest "POST"
+            "https://example.com/ok"
+            (Test.Http.expectJsonBody
+                (Json.Decode.field "version" Json.Decode.string)
+                (Expect.equal "3.1.5")
+            )
+        |> ...
+
+-}
+expectJsonBody :
```

Comment:
This seems like a useful addition! What made you add it? Does it need a test?

Reply:
I was originally going to use it in the test example in the guide, but after writing it, it just seemed to make the guide more confusing

Reply:
I added it here to simplify some of the merging of the other in-progress branches.  I can add a test though

Reply:
Added in 807fcdd

Reply:
Great!